Library/Homebrew/cleanup.rb

```diff
@@ -50,7 +50,7 @@ def self.cleanup_cache
           cleanup_path(path) { path.unlink }
           next
         end
-        if path.basename.to_s == "java_cache" && path.directory?
+        if %w[java_cache npm_cache].include?(path.basename.to_s) && path.directory?
```

Comment:
Possibly duplicating the existing `java_cache` block for `npm_cache` as well. Let's see what Martin has to say though, it's possible my memory is failing me, it happens alarmingly often.